src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -78,7 +78,22 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
 
     componentDidMount() {
         this.fetchFile();
-        this.fetchData();
+        this.fetchAccessStats();
+        this.fetchClassification();
+    }
+
+    componentDidUpdate(prevProps: Props) {
+        const { hasAccessStats, hasClassification } = this.props;
+        // Component visibility props such as hasAccessStats can sometimes be flipped after an async call
+        const hasAccessStatsVisibilityChanged = prevProps.hasAccessStats !== hasAccessStats;
```

Comment:
Minor: It seems inconsistent to add "Visibility" to these variables; seems like it should just be `hasAccessStatsChanged`.